.travis.yml

```diff
@@ -1,29 +1,50 @@
 language: cpp
-
-os: osx
-
-osx_image: xcode10.3
-
-git:
-  depth: false
-  quiet: true
+jobs:
+  include:
+    - os: osx
+      osx_image: xcode10.3
+      script: 
+        - cd build
+        - cmake -DBINARY_RELEASE=ON ..
+        - sudo cmake --build . --target package -j $(sysctl -n hw.physicalcpu)
+    - os: linux
+      dist: bionic
+      arch: ppc64le
+      script: 
+        - cd build
+        - cmake -DBINARY_RELEASE=ON ..
+        - sudo cmake --build .

```

Comment:
@AJenbo What is the flag to package binary into a tarball?

Reply:
Packaging has only been configured for Mac OS, all other releases are done by yours truely

Reply:
I see. So for every release, you have to cut releases manually? I assume so because I could not see any script/command that handle packaging in CircleCI pipeline. I could bake a script to package the tarball but I think it would be better to re-use the same script that other linux arches are on

Reply:
We plan to use CPack: https://github.com/diasurgical/devilutionX/issues/212